draft-js-emoji-plugin/yarn-error.log

```diff
@@ -0,0 +1,15672 @@
+Arguments: 
```

Comment:
This file shouldn't go into the repo. We should add this file to .gitignore to avoid it committing accidentally in the future.